velox/common/memory/tests/ByteStreamTest.cpp

```diff
@@ -431,6 +431,44 @@ TEST_P(InputByteStreamTest, inputStream) {
   ASSERT_TRUE(byteStream->atEnd());
 }
 
+TEST_P(InputByteStreamTest, iobuf) {
+  if (!GetParam()) {
+    return;
+  }
+  const auto streamSize = 4096;
+  std::vector<ByteRange> byteRanges;
+  std::uint8_t buffer[streamSize];
+  for (auto i = 0; i < streamSize; ++i) {
+    buffer[i] = i % 256;
+  }
+  byteRanges.push_back(ByteRange{buffer, streamSize, 0});
+
+  std::uint8_t buffer2[streamSize];
+  for (auto i = 0; i < streamSize; ++i) {
+    buffer2[i] = i % 13;
+  }
+  byteRanges.push_back(ByteRange{buffer2, streamSize, 0});
+
+  auto byteStream = createStream(byteRanges);
+  auto bufferStream = dynamic_cast<BufferInputStream*>(byteStream.get());
+  for (int offset = 0; offset < streamSize;) {
+    auto iobuf = bufferStream->readBytes(streamSize / 8);
```

Comment:
We are not testing the case one IOBuf chain cross 2 buffers.  Let's cover that as well.

Reply:
added.